frontend/components/sidebar/left/SidebarLeftFooter.vue

```diff
@@ -199,103 +100,124 @@ const closeOtherMenus = (id: number) => {
 
 const sidebar = useSidebar();
 
-const createButtons: MenuSelector[] = [
-  {
-    id: 1,
-    label: "components._global.new-event",
-    routeURL: "/events/create",
-    iconURL: "bi:calendar-check",
-    selected: false,
-  },
-  {
-    id: 2,
-    label: "components.sidebar-left-selector.label.new-organization",
-    routeURL: "/organizations/create",
-    iconURL: "IconOrganization",
-    selected: false,
-  },
-  {
-    id: 3,
-    label: "components._global.new-group",
-    routeURL: "/",
-    iconURL: "IconGroup",
-    selected: false,
-  },
-  {
-    id: 4,
-    label: "components._global.new-resource",
-    routeURL: "/resources/create",
-    iconURL: "IconResource",
-    selected: false,
-  },
-];
-
-const infoButtons: MenuSelector[] = [
-  {
-    id: 1,
-    label: "_global.help",
-    routeURL: "/help",
-    iconURL: "bi:question-circle",
-    selected: false,
-  },
-  {
-    id: 2,
-    label: "_global.documentation",
-    routeURL: "/docs",
-    iconURL: "bi:layout-text-sidebar-reverse",
-    selected: false,
-  },
-  {
-    id: 3,
-    label: "_global.legal",
-    routeURL: "/legal",
-    iconURL: "IconLegal",
-    selected: false,
-  },
-];
-
-const userButtons: MenuSelector[] = [
-  {
-    id: 1,
-    label: "components.sidebar-left-selector.label.your-profile",
-    routeURL: "/",
-    iconURL: "bi:person-circle",
-    selected: false,
-  },
-  {
-    id: 2,
-    label: "components.sidebar-left-selector.label.your-events",
-    routeURL: "/",
-    iconURL: "bi:calendar-check",
-    selected: false,
-  },
-  {
-    id: 3,
-    label: "components.sidebar-left-selector.label.your-orgs",
-    routeURL: "/",
-    iconURL: "IconOrganization",
-    selected: false,
-  },
+const menu: BarMenu[] = [
```

Comment:
@andrewtavis, what can you advise?

Reply:
Confirmed that having it all in the mobile burger menu will be the case. If we migrate this code to a common place, I'd be happy to put it into the right sidebar afterwards. Or @tatianakharitonovich I can also make an issue for this and you could implement?

Reply:
> Confirmed that having it all in the mobile burger menu will be the case. If we migrate this code to a common place, I'd be happy to put it into the right sidebar afterwards. Or @tatianakharitonovich I can also make an issue for this and you could implement?

Yes!I suppose I could implement it!

Reply:
> Ok, making the issue now :) Please let me know if you'd like to refactor this code out as a part of this PR or the one for the new issue. That'll then determine a final review here 😊

I would like to refactor this code out for the new issue

Reply:
Sounds good! I've made #663 for this :)

Reply:
I'll do the review later on this evening after work, @tatianakharitonovich. Thanks for the efforts here!

Reply:
Howdy, some thoughts about it:
- Those "menu options" seem to answer to an application configuration, and should be implementation agnostic. We could have a folder named `configurations` or `settings` with `.json` and a controller ("working as a repository") that exposes those data structures with their type definition.
- I think that this should come as a prop for this component and the client's components just ask the controller for that configuration information, passing it as a prop.
- I'm worried about the `selected` property, this could be handled by the client's components using an event like `selected`. My concern here is related to the responsibility, I think that the option configuration should not hold reactive state like `selected`, it should be two different things.

None of the above are mandatory 😂, I think that we could take those decisions later and test ideas about it, pretty sure that this "pattern" could be used in other areas of the FrontEnd app, improving the developing experience, we could start with this and see how it goes.

@tatianakharitonovich I'd be glad to help you working on the refactor if you want me ☺️

Reply:
Are we good to merge here then you two can collaborate on #663?

Reply:
Post review from me. Just checking that you're fine to work with the current state, @rigojr :)

Reply:
Fine by me, let's merge it

Reply:
I just want to sync if I should make any additions to this PR or can I work on issue https://github.com/activist-org/activist/issues/663 in another branch?

Reply:
Oh, I am sorry, I just understood.
I am good to merge here then we can collaborate on https://github.com/activist-org/activist/issues/663 :)

Reply:
Yes feel free to start on the new issue, and I'll do a final check and bring this in soon :) Maybe not tonight, but tomorrow at the latest.